### PR TITLE
[network-status] Flask project to show network status

### PR DESCRIPTION
**Pull request by janet-harmony**
Sample Page:
Screen Shot 2020-02-24 at 3 36 44 PM

### CODE REVIEW

File: monitoring/status-page/status/routes.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from status import app
+
+watchdog = 'http://watchdog.hmny.io/status-%s'
+network_list = [('Mainnet', 'mainnet'),('LRTN', 'testnet'), ('OSTN', 'staking')]
+
+@app.route('/status')
+def status():
+    statuses = {}
+    for n in network_list:
+        r = requests.get(watchdog % n[1], auth=('harmony', 'harmony.one'))
```
Comment (LeoHChen) [inline monitoring/status-page/status/routes.py]: no need to do the authentication, it should be public.

File: monitoring/status-page/status/routes.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import argparse
+import json
+import requests
+from flask import render_template
+from status import app
+
+watchdog = 'http://watchdog.hmny.io/status-%s'
```
Comment (LeoHChen) [inline monitoring/status-page/status/routes.py]: do you need to hardcode the URL?  consider to use reverse proxy by nginx as well, like we did for watchdog.

Comment (janet-harmony) [reply monitoring/status-page/status/routes.py]: The URL is for getting the network status data from existing Watchdog services, not for serving the webpage.

File: monitoring/status-page/status/routes.py (hunk 1 above)
Comment (janet-harmony) [reply monitoring/status-page/status/routes.py]: Authentication is for getting the data from existing Watchdog services.

**Pull request merged**